### PR TITLE
Feature/aapd 264 fix

### PR DESCRIPTION
**Pull request by chrishawldev**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-264

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/pdf-service-api/Dockerfile
Comment (bryn-pins) [inline packages/pdf-service-api/Dockerfile:1]: local file commit

Comment (chrishawldev) [reply packages/pdf-service-api/Dockerfile:1]: Ah well spotted, nice one

File: packages/forms-web-app/src/dynamic-forms/dynamic-components/list-add-more/index.njk [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <div class="{{question.width}}">
       {# TODO update to persist main question header text #}
       
-      <p class="govuk-heading-l">{{question.question}}</p>
+      <p class="govuk-heading-l">{{question.pageTitle}}</p>
```
Comment (bryn-pins) [inline packages/forms-web-app/src/dynamic-forms/dynamic-components/list-add-more/index.njk:16]: Looking at this, should this be an h1 and isPageHeading be set to false in the field set.
Or is it this way round on purpose?

**Review by bryn-pins — APPROVED** [review]
Happy to approve although have a question about the H1, not part of this change though

**Pull request merged**